hikari/impl/entity_factory.py

```diff
@@ -299,7 +299,7 @@ def deserialize_application(self, payload: data_binding.JSONObject) -> applicati
             owner=self.deserialize_user(payload["owner"]),
             rpc_origins=payload["rpc_origins"] if "rpc_origins" in payload else None,
             summary=payload["summary"],
-            verify_key=bytes(payload["verify_key"], "utf-8") if "verify_key" in payload else None,
+            verify_key=bytes.fromhex(payload["verify_key"]) if "verify_key" in payload else None,
```

Comment:
Don't really understand this change

Reply:
why not? It wasn't handling it properly before

Reply:
Actually, checking in the docs, this should just be a string :thinking:

Reply:
Except the docs state in the section for this object that this is base64 (which is wrong) encoded bytes and in other places state that it should be handled as hexed bytes

Reply:
btw this field is also known as the application's public key and it might be worth renaming it since erm it's only ever called verify key here

Reply:
Hmmm, docs are weird for this, so ima just trust you on this one. As per the rename, sure, go for it.

Reply:
Just renamed it to "public_key" and fixed our docs ~~which were also wrong cause copied from discord's docs~~